pkg/controller/migplan/validation.go

```diff
@@ -978,20 +980,20 @@ func (r ReconcileMigPlan) validateDestinationNamespaces(plan *migapi.MigPlan) er
 		})
 		return nil
 	}
-	longNamespaceNames := []string{}
+	nonDNSCompliantNames := []string{}
 	for _, ns := range namespaces {
-		if len(ns) > 63 {
-			longNamespaceNames = append(longNamespaceNames, ns)
+		if errs := validation.IsDNS1035Label(ns); len(errs) != 0 {
+			nonDNSCompliantNames = append(nonDNSCompliantNames, ns)
```

Comment:
I'm wondering whether we should at least log these errors returned. It's probably too cumbersome to include them in the condition itself, but if we logged them, a user could look at the log if they didn't understand why they got the validation warning.